app/models/tenant.rb

```diff
@@ -33,6 +33,7 @@ class Tenant < ApplicationRecord
   has_many :services, :dependent => :destroy
 
   belongs_to :default_miq_group, :class_name => "MiqGroup", :dependent => :destroy
+  belongs_to :source, :polymorphic => true
```

Comment:
I'm not so sure about the naming here, because most of our polymorph relations use `resource` as the name.
@chrisarcand thoughts?